loadtesting/hae_tunnistautuneena.js

```diff
@@ -0,0 +1,110 @@
+import { browser } from 'k6/experimental/browser';
+import { check } from 'k6';
```

Comment:
Kun ajoin tätä lokaalisti niin tuli seuraava ongelma (käsin flow kyllä näyttää menevän läpi):

INFO[0003] initial load doneOpintopolku – hakulomake – Testilomake (ht)  source=console
INFO[0003] initial load doneOpintopolku – hakulomake – Testilomake (ht)  source=console
INFO[0004] "Handle oppija session fetch, sess data{:logged-in false}"  browser_source=console-api source=browser
INFO[0004] "Handle oppija session fetch, sess data{:logged-in false}"  browser_source=console-api source=browser
ERRO[0007] Failed to load resource: the server responded with a status of 404 ()  browser_source=network line_number=0 source=browser stacktrace="<nil>" url="https://testi.apro.tunnistus.fi/disruption/disruption_message.json"
ERRO[0007] Failed to load resource: the server responded with a status of 404 ()  browser_source=network line_number=0 source=browser stacktrace="<nil>" url="https://testi.apro.tunnistus.fi/disruption/disruption_message.json"
INFO[0008] hetu input done: 210281-9988                  source=console
INFO[0008] click suomiFiTunnistauduButton Testitunnistaja  source=console
INFO[0008] hetu input done: 210281-9988                  source=console
INFO[0008] click suomiFiTunnistauduButton Testitunnistaja  source=console
INFO[0010] Error in suomiFiTunnistauduButton {"value":{}}  source=console
INFO[0010] Error in suomiFiTunnistauduButton {"value":{}}  source=console
INFO[0011] Waiting for email input...                    source=console
INFO[0011] Waiting for email input...                    source=console
ERRO[0012] Uncaught (in promise) GoError: waiting for selector "input[data-test-id=\"email-input\"]": execution context changed; most likely because of a navigation
ui     [at github.com/grafana/xk6-browser/browser.mapPage.func13 (native)
	at file:///Users/sankor/workspace/ataru/loadtesting/hae_tunnistautuneena.js:59:23(177)  executor=shared-iterations scenario=ui
ERRO[0012] Uncaught (in promise) GoError: waiting for selector "input[data-test-id=\"email-input\"]": execution context changed; most likely because of a navigation
ui     [at github.com/grafana/xk6-browser/browser.mapPage.func13 (native)
	at file:///Users/sankor/workspace/ataru/loadtesting/hae_tunnistautuneena.js:59:23(177)  executor=shared-iterations scenario=ui